apps/scout-agents/src/main.ts

```diff
@@ -0,0 +1,39 @@
+import { log } from '@charmverse/core/log';
+import { setBotCommands } from '@packages/telegram/setBotCommands';
+import { setTelegramBotWebhook } from '@packages/telegram/setTelegramBotWebhook';
+
+import { SCOUT_AGENT_BUILDER_TELEGRAM_BOT_TOKEN } from './agents/constants';
+import { TELEGRAM_API_PATH } from './api/builder-agent/telegram';
+import { app } from './server';
+
+const host = process.env.HOST ?? 'localhost';
+const port = process.env.PORT ? Number(process.env.PORT) : 3000;
+
+app.listen(port, host, () => {
+  log.info(`[ ready ] http://${host}:${port}`);
+});
+
+async function setupTelegramWebhook() {
+  try {
+    await setTelegramBotWebhook({
+      token: SCOUT_AGENT_BUILDER_TELEGRAM_BOT_TOKEN,
+      endpoint: `${process.env.DOMAIN}${TELEGRAM_API_PATH}`
+    });
+    await setBotCommands({
+      token: SCOUT_AGENT_BUILDER_TELEGRAM_BOT_TOKEN,
+      configuration: {
+        commands: [
+          {
+            command: '/build',
+            description: 'Join Scout Game as a developer'
+          }
+        ]
+      }
+    });
+    log.info('Telegram bot webhook set');
+  } catch (error) {
+    log.error('Error setting telegram bot webhook', { error });
+  }
+}
+
+setupTelegramWebhook();
```

Comment:
kinda hacky that this runs on every app start

Reply:
The core telegram bot identifier is the API token.

Setting the webhook is idempotent, so if you send the the same value as current it's totally fine.

I've put this here for simplicity so we can just run the app and have it setup telegram without extra config